.golangci.yml

```diff
@@ -15,6 +15,8 @@ linters-settings:
   goconst:
     min-len: 2
     min-occurrences: 2
+  depguard:
+    list-type: blacklist
```

Comment:
What does it mean?

Reply:
So depguard is based off a whitelist or a blacklist. If nothing is passed it, it defaults to a whitelist with your current packages in it. Since this project has many imports, and empty blacklist will suffice (since it will allow everything). I could disable the linter if you would prefer?

Reply:
Some users run `golangci-lint` like we do: `golangci-lint run --enable-all`, and it shouldn't blacklist anything if not explicitly configured. Remove this configuration please and ensure that we don't need to disable linter explicitly after that. Also, linter should do nothing if it finds that nothing was configured.

Reply:
So change the default to an empty blacklist if no configuration is passed?

Not sure I understand the reasoning behind not running a linter if it is enabled but not configured. Should it not have a default configuration (empty blacklist which allows everything)? Which brings up a good point I can short cut the code in that case.